.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "python.pythonPath": "venv/bin/python",
+  "python.pythonPath": "/mnt/c/Users/ander/Documents/CMPUT404/project/venv/bin/python3.6",
```

Comment:
I'll change this line to `"python.pythonPath": "/venv/bin/python3",` because your editor changed the path. Everyone should have python3 in their venv even if the subversion is different.